src/config.rs

```diff
@@ -3,7 +3,10 @@ use dirs;
 use serde::Deserialize;
 use std::env;
 use std::fs;
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
+
+#[cfg(any(target_os = "linux", target_os = "macos"))]
+use std::path::Path;
```

Comment:
Please note that this fixes a warning of unused import when compiling not on linux || macos.